app/services/power_manager/virt.rb

```diff
@@ -35,12 +35,9 @@ def default_action(action)
 
     def action_map
       super.deep_merge({
-                         :on       => 'start',
-                         :off      => 'stop',
-                         :soft     => 'reboot',
-                         :cycle    => 'reset',
-                         :status   => {:action => :virt_state, :output => :state_output},
-                         :state    => {:action => :virt_state, :output => :state_output},
+                         :poweroff => 'soft',
+                         :status   => {:action => :virt_state, :output => :state_output, :default => nil},
+                         :state    => {:action => :virt_state, :output => :state_output, :default => nil},
                        })
```

Comment:
Don't understand why on/off/cycle is being dropped.

Reply:
it's just removed from the mapping table, because after the refactor the names match from the symbols in REAL_ACTION and the virt backend.

So:

start == start
stop == stop

that's why on / off is dropped, and cycle was my mistake to assume it's actually reboot!

=> but as you have provided already a pretty good expected behaviour I will go over it, rework what I did to match your proposal, and then we can take a look at it again.